### ISSUE TITLE
qualifier usage for a relationship

### ISSUE THREAD

**Issue by yihect** [umlet/umlet#420]
When I use 'q1=userID' as a qualifier, it doesn't work.
   String 'q1=userID' is showed as an association name in the attached pictures.

![q_text](https://cloud.githubusercontent.com/assets/5461855/21713775/8de08456-d436-11e6-99ef-af558870ba9b.png)

![q_diag](https://cloud.githubusercontent.com/assets/5461855/21713785/9b4b9d1a-d436-11e6-8b16-388fd1a63a42.png)

  My environment is umlet14.2(standalone) + Ubuntu 14.04.4 LTS.

**Comment by afdia** [comment]
Try `lt=[userID]-`

**Comment by yihect** [comment]
Yeah, but the key rectangle is placed across the edge of the source object. 
 It's better to align it with the edge, more in line with UML spec, doesn't it? 
 
![qshould](https://cloud.githubusercontent.com/assets/5461855/21756064/2c3d3532-d657-11e6-8a34-dcf2612122ac.png)

**Comment by afdia** [comment]
As far as I know both variants are required by different UML diagram types (see #220), but you're right that currently the edge-aligned variant is not possible with the new relation element and old relation element only supports the non-edge-aligned variant.

As mentioned in #220 this will be made configurable via the text command in the future.
